src/test/librust/sapling_rpc_wallet_tests.cpp

```diff
@@ -21,6 +23,40 @@
 
 extern UniValue CallRPC(std::string args); // Implemented in rpc_tests.cpp
 
+// Remember: this method will be moved to an utility file in the short future. For now, it's in sapling_keystore_tests.cpp
+extern libzcash::SaplingExtendedSpendingKey GetTestMasterSaplingSpendingKey();
+
+namespace {
+
+    /** Set the working directory for the duration of the scope. */
+    class PushCurrentDirectory {
+    public:
+        PushCurrentDirectory(const std::string &new_cwd)
+                : old_cwd(boost::filesystem::current_path()) {
+            boost::filesystem::current_path(new_cwd);
+        }
+
+        ~PushCurrentDirectory() {
+            boost::filesystem::current_path(old_cwd);
+        }
+    private:
+        boost::filesystem::path old_cwd;
+    };
+
+}
+
+void CheckRPCThrows(std::string rpcString, std::string expectedErrorMessage) {
+    try {
+        CallRPC(rpcString);
+        // Note: CallRPC catches (const UniValue& objError) and rethrows a runtime_error
+        BOOST_FAIL("Should have caused an error");
+    } catch (const std::runtime_error& e) {
+        BOOST_CHECK_EQUAL(expectedErrorMessage, e.what());
+    } catch(const std::exception& e) {
+        BOOST_FAIL(std::string("Unexpected exception: ") + typeid(e).name() + ", message=\"" + e.what() + "\"");
+    }
+}
```

Comment:
It seems that this function is not being called anywhere (`BOOST_CHECK_THROW` is used instead). Is it coming in a later refactoring?

Reply:
nope, good catch. Cleaned.